2023-01-30 15:55:26 database: opening /home/ingo/git/thesis/ota-tester-pollingIntervalls/.metals/metals
org.h2.message.DbException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:46451)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:46451)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.convert(DbException.java:383)
	at org.h2.engine.Database.startServer(Database.java:674)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:128)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:152)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:46451)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:46451)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.tools.Server.start(Server.java:540)
	at org.h2.engine.Database.startServer(Database.java:672)
	... 27 more
2023-01-30 15:55:46 database: opening /home/ingo/git/thesis/ota-tester-pollingIntervalls/.metals/metals
org.h2.message.DbException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:38377)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:38377)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.convert(DbException.java:383)
	at org.h2.engine.Database.startServer(Database.java:674)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:38377)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:38377)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.tools.Server.start(Server.java:540)
	at org.h2.engine.Database.startServer(Database.java:672)
	... 27 more
2023-01-30 15:56:05 database: opening /home/ingo/git/thesis/ota-tester-pollingIntervalls/.metals/metals
org.h2.message.DbException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:34745)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:34745)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.convert(DbException.java:383)
	at org.h2.engine.Database.startServer(Database.java:674)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:128)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:152)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:34745)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:34745)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.tools.Server.start(Server.java:540)
	at org.h2.engine.Database.startServer(Database.java:672)
	... 28 more
2023-01-30 15:56:24 database: opening /home/ingo/git/thesis/ota-tester-pollingIntervalls/.metals/metals
org.h2.message.DbException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:40101)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:40101)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.convert(DbException.java:383)
	at org.h2.engine.Database.startServer(Database.java:674)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Fehler beim Öffnen von Port "H2 TCP Server (tcp://localhost:40101)" (Port wird ev. bereits verwendet), Grund: "timeout; please check your network configuration, specially the file /etc/hosts"
Exception opening port "H2 TCP Server (tcp://localhost:40101)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.tools.Server.start(Server.java:540)
	at org.h2.engine.Database.startServer(Database.java:672)
	... 28 more
2023-01-30 15:56:25 database: opening /home/ingo/git/thesis/ota-tester-pollingIntervalls/.metals/metals
org.h2.message.DbException: Fehler beim Öffnen der Datenbank: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:315)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:128)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:152)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.clearAll(Fingerprints.scala:23)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:67)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Fehler beim Öffnen der Datenbank: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:504)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 33 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 30 more
